app/views/content/blog/overcoming-challenges-to-become-a-teacher-autism.md

```diff
@@ -44,4 +44,6 @@ I am a little concerned about disclosing the fact that I am autistic, as I didn
 
 My answer would be: only if it’s right for you! It’s certainly not for everyone, but if you feel pulled towards it (or if you feel like something inside you is pushing you towards it despite your best efforts!) then go for it. Don’t worry about whether or not you will be able to gain a place on a course, as there is a lot of help out there. Your own ambition should be your only limit.
 
+Find out about the [support you can get while training to teach if you're disabled](/funding-and-support/if-youre-disabled).
```

Comment:
Do you think the links should go here in the main content, or in the indented text at the end? 
![image](https://user-images.githubusercontent.com/94541493/198060688-901d5571-96c8-40be-aa85-cdcaf37a70f6.png)

Reply:
I agree - I think this would be better in the footnote, otherwise it sounds like it is something that the author is saying (rather than additional supporting information from us)